packages/components/src/components/vertical-tab/vertical-tab-content-container.tsx

```diff
@@ -51,10 +51,15 @@ const SideNotes = ({ class_name, side_notes }: TSideNotes) => {
 };
 
 const ContentWrapper = ({ children, has_side_note }: React.PropsWithChildren<TContentWrapper>) => {
-    if (has_side_note) {
-        return <div className='dc-vertical-tab__content-inner'>{children}</div>;
-    }
-    return children as JSX.Element;
+    return (
```

Comment:
The previous implementation generated entirely new markup based on the `has_side_note` prop, causing all routes to remount and triggering the same tracking events multiple times.